jdaviz/configs/specviz2d/tests/test_parsers.py

```diff
@@ -33,13 +33,14 @@ def test_2d_parser_jwst(specviz2d_helper):
 
     # Also check the coordinates info panel.
     viewer_2d = specviz2d_helper.app.get_viewer('spectrum-2d-viewer')
-    label_mouseover = specviz2d_helper.app.session.application._tools['g-coords-info']
+    label_mouseover = specviz2d_helper._coords_info
     label_mouseover._viewer_mouse_event(viewer_2d,
                                         {'event': 'mousemove', 'domain': {'x': 350, 'y': 30}})
     assert label_mouseover.as_text() == ('Pixel x=0350.0 y=0030.0 Value +3.22142e+04 MJy / sr',
                                          'Wave 6.24985e+00 um', '')
 
 
+@pytest.mark.xfail(reason="Transpose not implemented for spectrum1d input")

```

Comment:
The current implementation drops support for passing `ext` or `transpose` when the input file can be parsed directly by specutils (as it defaults to specutils over reading the fits HDUList).  Do we need to keep this case?  If so, we would need to either:
* implement support for extension and/or transpose in specutils and expose those options in the specutils parser
* always send spectra through the hdulist parser, instead of specutils
* implement transpose _after_ getting the spectrum1d object
* allow both fits and specutils parsers to run and decide based on priority or kwarg or passed kwargs (if extension or transpose is passed, then choose the fits parser over the specutils parser)

Reply:
NOTE: we do have a follow-up ticket for this - this does affect specviz2d, so we likely want to address this before the next release.

Reply:
#3549 hopes to unskip this test 🤞